### DISCUSSION TITLE
not starting

### DISCUSSION THREAD

**Discussion by Hynesman** [google/schedviz#37] [General]
Hi,
I can't seem to get it up and running.
I've got a fresh ubuntu install and made sure yarn was installed.

When i try to start the server i get this

`yarn bazel run server -- -- -storage_path="."
yarn run v1.22.17
warning From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
$ /home/mike/schedwiz/schedviz/node_modules/.bin/bazel run server -- -storage_path=.
INFO: Writing tracer profile to '/home/mike/.cache/bazel/_bazel_mike/05994261877ce60cd0986d24ffe1df51/command.profile.gz'
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: //client/app/sidebar:sidebar: no such attribute 'strict_templates' in 'ng_module' rule
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/settings_menu/BUILD.bazel:8:1: //client/app/sidebar/settings_menu:settings_menu: no such attribute 'strict_templates' in 'ng_module' rule
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/thread_table/BUILD.bazel:8:1: //client/app/sidebar/thread_table:thread_table: no such attribute 'strict_templates' in 'ng_module' rule
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: Target '//client/app/sidebar/settings_menu:settings_menu' contains an error and its package is in error and referenced by '//client/app/sidebar:sidebar'
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: Target '//client/app/sidebar/thread_table:thread_table' contains an error and its package is in error and referenced by '//client/app/sidebar:sidebar'
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: Target '//client/app/sidebar:sidebar.ts' contains an error and its package is in error and referenced by '//client/app/sidebar:sidebar'
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: Target '//client/app/sidebar:sidebar_module.ts' contains an error and its package is in error and referenced by '//client/app/sidebar:sidebar'
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: Target '//client/app/sidebar:sidebar.ng.html' contains an error and its package is in error and referenced by '//client/app/sidebar:sidebar'
ERROR: /home/mike/schedwiz/schedviz/client/app/sidebar/BUILD.bazel:8:1: Target '//client/app/sidebar:sidebar.css' contains an error and its package is in error and referenced by '//client/app/sidebar:sidebar'
ERROR: /home/mike/schedwiz/schedviz/client/app/dashboard/BUILD.bazel:8:1: Target '//client/app/sidebar:sidebar' contains an error and its package is in error and referenced by '//client/app/dashboard:dashboard'
ERROR: Analysis of target '//server:server' failed; build aborted: Analysis failed
INFO: Elapsed time: 5.942s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (212 packages loaded, 11271 targets configure\
FAILED: Build did NOT complete successfully (212 packages loaded, 11271 targets configure\
d)
    Fetching ..._tools; Cloning 589c23e65e65055d47b9ad4a99723bc389136265 of https://go.go\
oglesource.com/tools
    Fetching ...leapis; Cloning 41d72d444fbe445f4da89e13be02078734fb7875 of https://githu\
b.com/googleapis/googleapis
    Fetching ...otobuf; Cloning 582743bf40c5d3639a70f98f183914a2c0cd0680 of https://githu\
b.com/protocolbuffers/protobuf
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`

**Comment by sabarabc** [comment]
Can you try running it at this commit: https://github.com/google/schedviz/tree/6c1c10eb4b9d77821c10cee41c32ba98288948e1

**Reply by Hynesman** [reply]
yes this helped.  thanks!
